extensions/data-plane/data-plane-framework/src/main/java/org/eclipse/dataspaceconnector/dataplane/framework/DataPlaneFrameworkExtension.java

```diff
@@ -83,24 +80,22 @@ public void initialize(ServiceExtensionContext context) {
         var workers = context.getSetting(WORKERS, DEFAULT_WORKERS);
         var waitTimeout = context.getSetting(WAIT_TIMEOUT, DEFAULT_WAIT_TIMEOUT);
 
+        var store = new InMemoryDataPlaneStore(IN_MEMORY_STORE_CAPACITY);
```

Comment:
The in-memory store should be pluggable, i.e. if a store implementation is not registered during initiaize, the in-memory one will be used in start()

Reply:
Implemented